docs/reference/errors-and-warnings/NU3043.md

```diff
@@ -0,0 +1,43 @@
+---
+title: NuGet Warning NU3043
+description: NU3043 warning code
+author: kartheekp-ms
+ms.date: 09/11/2024
+ms.topic: reference
+f1_keywords: 
+  - "NU3043"
+---
+
+# NuGet Warning NU3043
+
+> Invalid value for `--certificate-fingerprint` option in the `dotnet nuget sign` command or the `CertificateFingerprint` option in the `NuGet.exe sign` command.
+The value must be a SHA-256, SHA-384, or SHA-512 certificate fingerprint (in hexadecimal).
+
+> This warning will be promoted to an error around the `.NET 10` timeframe.
```

Comment:
.NET 10 isn't a code snippet, class name, etc, so I don't think that putting it in a code block feels right.

Reply:
Fixed in the latest commit.